openbb_terminal/stocks/stocks_controller.py

```diff
@@ -197,7 +197,7 @@ def call_search(self, other_args: List[str]):
             help="Search by sector to find stocks matching the criteria",
         )
         parser.add_argument(
-            "-g",
+            "--ig",
```

Comment:
no need for both.  Can just have industry-group.  Same for below.

Reply:
`industry-group` and `exchange-country` do not list/autosuggest the options. Any suggestions on how to resolve this?

Reply:
Am not sure why.  I pushed a proposed solution just calling it industrygroup and exchangecountry instead of any separator.  Seems to work.  Let me know if thats all good

Reply:
Looks good to me. I will modify the tests and push the changes.

Reply:
Tests are good. Everything works!